### ISSUE TITLE
The CXX compiler identification is unknown.

### ISSUE THREAD

**Issue by yanhui09** [bioconda/bioconda-recipes#42814]
I tried to make one recipe for one bioinformatic tool [meshclust3](https://github.com/BioinformaticsToolsmith/Identity) but it couldn't identify CXX compiler in local build. I am new to bioconda recipe and got stuck. It would be great if anyone could give a hand.

[bash.sh](https://github.com/yanhui09/bioconda-recipes/blob/meshclust3/recipes/meshclust3/build.sh)
[meta.yaml](https://github.com/yanhui09/bioconda-recipes/blob/meshclust3/recipes/meshclust3/meta.yaml)

```
bioconda-utils build --docker --mulled-test --packages meshclust3
```
log:
```
-- The C compiler identification is GNU 12.3.0
-- The CXX compiler identification is unknown
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: $BUILD_PREFIX/bin/x86_64-conda-linux-gnu-cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
CMake Error at CMakeLists.txt:5 (project):
  The CMAKE_CXX_COMPILER:

    g++-7

  is not a full path and was not found in the PATH.

  Tell CMake where to find the compiler by setting either the environment
  variable "CXX" or the CMake cache entry CMAKE_CXX_COMPILER to the full path
  to the compiler, or to the compiler name if it is in the PATH.
```

**Comment by yanhui09** [comment]
https://github.com/BioinformaticsToolsmith/Identity/issues/17#issue-1879220255

**Comment by yanhui09** [comment]
v2.0 has some bugs in CMakeLIsts.txt which has fixed requirement for gcc-7, but I mixed this version with the one on the master branch. I need to suggest @hani-girgis a new tag for the one on the master branch, since `meshclust3` was not included in v2.0 yet.
https://github.com/BioinformaticsToolsmith/Identity/blob/e8a8f96821480eb1c66028e57eed7d1babea282f/CMakeLists.txt#L3
